### DISCUSSION TITLE
Wasnt trpc 11 supported?

### DISCUSSION THREAD

**Discussion by alanxp** [icflorescu/trpc-sveltekit#115] [Q&A]
I Previously asked about support for trpc 11, however it doesnt seem to work since this is deprecated
![image](https://github.com/icflorescu/trpc-sveltekit/assets/15839818/46e80ed1-b1b6-4c5d-9ecf-cd6f825324c7)

**Comment by icflorescu** [comment]
Some of the types in the examples were indeed deprecated, however this doesn't mean tRPC v11 is not supported.
Please make sure you understand basic concepts before attempting to use the tRPC ecosystem. 

I have updated the examples - you can use these instead:

```ts
import type { RequestEvent } from '@sveltejs/kit';

export async function createContext(event: RequestEvent) {
  return {
    event // 👈 `event` is now available in your context
  };
}

export type Context = Awaited<ReturnType<typeof createContext>>;
```

Please note that tRPC v11 is still in beta.
